server/utils/formUtils.ts

```diff
@@ -0,0 +1,25 @@
+import type { ErrorMessages } from 'approved-premises'
+
+export default function dateFieldValues(
+  fieldName: string,
+  context: Record<string, unknown>,
+  errors: ErrorMessages = {},
+) {
+  return [
```

Comment:
Feel free to ignore as it's minor but I think it could be clearer to define `errors[fieldName] ? 'govuk-input--error' : ''` as a variable (EG `errorClass`) and use that on lines 10, 15 and 20 instead of duplicating the logic